mondey_backend/src/mondey_backend/routers/users.py

```diff
@@ -95,12 +95,14 @@ async def get_child_image(
         current_active_user: CurrentActiveUserDep,
         child_id: int,
     ):
-        child = child = get_db_child(session, current_active_user, child_id)
+        child = get_db_child(session, current_active_user, child_id)
         image_path = pathlib.Path(
             f"{app_settings.PRIVATE_FILES_PATH}/children/{child.id}.jpg"
         )
         if not image_path.exists():
-            raise HTTPException(404)
+            raise HTTPException(404, detail="Image not found")
+        if child.has_image is False:
+            raise HTTPException(404, detail="Child does not have an image")
```

Comment:
rather than raising an error, maybe we should have a default blank avatar image that we return if there isn't an image for the child?

Reply:
good point.

Reply:
I would prefer to do this later, however.